### PR TITLE
Add dev tools and teps to resources.md

### PR DESCRIPTION
**Pull request by UncleClapton**
**Summary of Changes**

Add links to Dga's devtools, and tips for developing.

**Screenshots (if necessary)**

**Additional context**

I have seen a lot of people who don't yet know about the tools we have available now. I made these changes so there's an easy place to go for this information.

### CODE REVIEW

File: docs/resources.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+
+### Symlink project files to your Community directory
+You can create a symlink (or "directory junction" in Windows terms) between your local repo and the game's community folder. This lets you develop from your local repository without having to copy files over when you're ready to test. Developers working on hot-reloadable files can get changes reflected in real-time, while still retaining a proper development environment
+
+First, be sure to remove any existing install of the mod from the game.
+
+Then in `cmd`, run the command below. Be sure to replace each path with the proper respective file path.
+
+```cmd
+mklink /J "C:\users\<user>\AppData\Microsoft Flight Simulator\Packages\Community\A32NX" "C:\path\to\cloned\repo\A32NX"
```
Comment (tyler58546) [inline docs/resources.md:27]: This path may be different for people who have a different version or set a custom install location.

File: docs/resources.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
 # Development Resources
 
+## Tools:
+- [WebUI-DevKit](https://github.com/dga711/msfs-webui-devkit) - In-game development overlay for html-ui content. Provides fast reload, console output, and more!
+- [devtools-backend-refurb](https://github.com/dga711/msfs-webui-devkit) - Chrome devtools server targeting Coherent GT. Gives you devtools for html-ui content. WARNING: This is still a work in progress, and tends to be very finnicky.
```
Comment (lukecologne) [inline docs/resources.md:5]:
```suggestion
- [devtools-backend-refurb](https://github.com/dga711/devtools-backend-refurb) - Chrome devtools server targeting Coherent GT. Gives you devtools for html-ui content. WARNING: This is still a work in progress, and tends to be very finnicky.
```
Duplicate link, probably meant this link

Comment (UncleClapton) [reply docs/resources.md]: uhh yeah... whoops!

File: docs/resources.md (hunk 1 above)
Comment (UncleClapton) [reply docs/resources.md:27]: I added it for example to where it could be. I'm not sure if there's really a better way to represent it.

**Review by AdenFlorian — APPROVED** [review]

**Pull request merged**